content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,256 @@
+- *Managed worker nodes*: Configure pools of nodes to meet your needs,
+then the provider makes sure those nodes are available and ready to implement
+upgrades when needed.
+- *Integration*: There are providers that integrate Kubernetes with other
+services you may need, such as storage, container registries, authentication
+methods, and development tools.
+
+Whether you build a production Kubernetes cluster yourself or work with
+partners, review the following sections to evaluate your needs as they relate
+to your cluster’s *control plane*, *worker nodes*, *user access*, and
+*workload resources*.
+
+## Production control plane
+
+The simplest Kubernetes cluster has the entire control plane and worker node
+services running on the same machine. You can grow that environment by adding
+worker nodes, as reflected in the diagram illustrated in
+[Kubernetes Components](/docs/concepts/overview/components/).
+If the cluster is meant to be available for a short period of time, or can be
+discarded if something goes seriously wrong, this might meet your needs.
+
+If you need a more permanent, highly available cluster, however, you should
+consider ways of extending the control plane. By default, one-machine control
+plane services are not highly available. If keeping the cluster up and running
+and ensuring that it can be repaired if something goes wrong is important,
+consider these steps::
+
+- *Choose deployment tools*: You can deploy a control plane using tools such
+as kubeadm, kops, and kubespray. Links from [Installing Kubernetes with deployment tools](/docs/setup/production-environment/tools/)
+contains tips for production-quality deployments using each of those deployment
+methods. Different [Container Runtimes](/docs/setup/production-environment/container-runtimes/)
+are available to use with your deployments.
+- *Manage certificates*: Secure communications between control plane services
+are implemented using certificates. These can be automatically generated
+during deployment or you can generate them using your own certificate authority.
+See [PKI certificates and requirements](/docs/setup/best-practices/certificates/) for details.
+- *Configure load balancer for apiserver*: External access to the cluster
+should be to a load balancer that can distribute requests to the apiserver service
+on different nodes. See 
+[Create an External Load Balancer](/docs/tasks/access-application-cluster/create-external-load-balancer/)
+for details.
+- *Separate and backup etcd service*: The etcd services can either run on the
+same machines as other control plane services or run on separate machines, for
+extra security and availability. Because etcd stores cluster configuration data,
+backing up the etcd database should be done regularly to ensure that you can
+repair that database if needed.
+An etcd cluster should have an [odd number of members](https://etcd.io/docs/next/faq/#why-an-odd-number-of-cluster-members)
```

Comment:
nit: I would leave the etcd details out and link to the etcd docs site?

Reply:
I'd suggest linking to `v3.4` rather than `next` here. `v3.4` is the latest stable version, whereas `next` is for upcoming versions (and is more likely to change, even though it's an FAQ page).

https://etcd.io/docs/v3.4/faq/#why-an-odd-number-of-cluster-members

Reply:
`/latest` is a good choice too.

Reply:
Okay. I changed it to a more general link to the etcd FAQ and used the suggested v3.4 link, omitting the specific section: "See the [etcd FAQ](https://etcd.io/docs/v3.4/faq/) for details on configuring and using etcd."